ksy_schema.json

```diff
@@ -243,7 +243,7 @@
         },
         "pad-right": {
           "type": "integer",
-          "description": "specify a byte which is the string or byte array padded with up to the total size"
+          "description": "specify a byte which is the string or byte array padded with after the end up to the total size\n\ncan be used only with \"size\" or \"size-eos: true\" (when the size is fixed)\n\nwhen \"terminator\":\n - isn't specified, then the \"pad-right\" controls where the string ends (basically acts like a terminator)\n - is specified, padding comes after the terminator, not before. The value is terminated immediately after the terminator occurs, so the \"pad-right\" has no effect on parsing and is only relevant for serialization"
```

Comment:
@dgelessus Are you satisfied with the description now? Would you change it in some way?

Reply:
This has all the important information now I think. :+1: